README.md

```diff
@@ -30,7 +30,7 @@ To custom the keywords and other stuff, <kbd>command</kbd> + <kbd>,</kbd> (Windo
 | todohighlight.isCaseSensitive  | boolean | true | Whether the keywords are case sensitive or not. |
 | todohighlight.keywords | array | N/A | An array of keywords that will be hilighted. You can also specify the style for each keywords here. See example below for more infomation. |
 | todohighlight.keywordsPattern  | string | N/A | Specify keywords via RegExp instead of `todohighlight.keywords` one by one. NOTE that if this presents, `todohighlight.keywords` will be ignored. And REMEMBER to escapse the back slash if there's any in your regexp (using \\ instead of signle back slash). |
-| todohighlight.defaultStyle | object | N/A | Specify the default style for custom keywords, if not specified, build in default style will be applied. [Checkout all the available styling properties](https://code.visualstudio.com/docs/extensionAPI/vscode-api#DecorationRenderOptions). |
+| todohighlight.defaultStyle | object | N/A | Specify the default style for custom keywords, if not specified, build in default style will be applied. [Checkout all the available styling properties](https://code.visualstudio.com/docs/extensionAPI/vscode-api#_a-namedecorationrenderoptionsaspan-classcodeitem-id280decorationrenderoptionsspann). |
```

Comment:
This's not working while the original one does work.

Reply:
Thats really strange... the link you've changed it back to does not work for me.

I made a fresh chrome profile and tried it, take a look. https://streamable.com/b65hw

Reply:
The longer one does not work because there is an extra `n`. It should be `_a-namedecorationrenderoptionsaspan-classcodeitem-id280decorationrenderoptionsspan`. When you hover the header, it will give you the `#` on the right which is the fragment identifier link that uses this longer one.

The shorter one does work for me in the latest Chrome, Firefox, and Edge and is even used as a fragment identifier link several times within that page -- check the source. Both work.

Reply:
I don't know what to say then. Tried it in Chrome, Firefox and Edge again, and same results as the video above.

I know thats the link you get from clicking on the hash, but `_a-namedecorationrenderoptionsaspan-classcodeitem-id280decorationrenderoptionsspan` just doesn't take me to the section.

Weird.

Reply:
@KamasamaK is right. There's an extra `n` in the tail.  Remove it then longer one works on my side.

Reply:
Ahh yes, removed the extra `n` and it worked. Pulled change with `n` removed.